.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Issue
+about: Report an issue, bug, or unexpected behavior.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the issue**
+Describe the issue that occurs and list steps to reproduce this behavior. Make sure to read [GRR server doesn't seem to run](https://grr-doc.readthedocs.io/en/latest/installing-grr-server/troubleshooting.html) and [I don't see my clients](https://grr-doc.readthedocs.io/en/latest/deploying-grr-clients/troubleshooting.html). If the errors occur after an upgrade, make sure to read [CHANGELOG](https://github.com/google/grr/blob/master/CHANGELOG) for possible reasons.
+
+**Error logs**
+```
+Copy-and-paste logs from the command you executed, GRR server log files, or flow logs.
+```
+
+**Environment**
```

Comment:
Would it make sense to move the environment before the *Describe the issue* section? Often this information is crucial to understand the issue.